Monitoring term style updates for 4.5 (#28070)
openshift/openshift-docs: modules/monitoring-setting-up-metrics-collection.adoc

[id="setting-up-metrics-collection_{context}"]
= Setting up metrics collection

> To use the metrics exposed by your service, you must configure OpenShift Monitoring to scrape metrics from the `/metrics` endpoint. You can do this using a `ServiceMonitor` custom resource definition (CRD) that specifies how a service should be monitored, or a `PodMonitor` CRD that specifies how a pod should be monitored. The former requires a `Service` object, while the latter does not, allowing Prometheus to directly scrape metrics from the metrics endpoint exposed by a pod.

Reviewer: ```suggestion
To use the metrics exposed by your service, you must configure OpenShift Monitoring to scrape metrics from the `/metrics` endpoint. You can do this by using a `ServiceMonitor` custom resource definition (CRD) that specifies how to monitor a service or a `PodMonitor` CRD that specifies how to monitor a pod. The former requires a `Service` object, while the latter does not, which allows Prometheus to directly scrape metrics from the metrics endpoint exposed by a pod.
```
